clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -781,6 +791,8 @@ bool isValidCIdentifier(StringRef S) {
 Error linkBitcodeFiles(SmallVectorImpl<OffloadFile> &InputFiles,
                        SmallVectorImpl<StringRef> &OutputFiles,
                        const ArgList &Args) {
+  if (Verbose)
+    llvm::errs() << "Linking bitcode files\n";
```

Comment:
Time-trace seems to be focused more on profiling than just tracing. It writes json to a file.
I guess, as a human user, I just wanted a simple way to see when clang-linker-wrapper was running LTO in-process.
Is there a third alternative that would work for you?

Reply:
That sounds more like debugging than verbose output, we could add something for it but in general I don't think it adds too much, since this path isn't really used anymore.